build/docfx.cmd

```diff
@@ -0,0 +1,17 @@
+SETLOCAL
```

Comment:
Is there a need to add a new top level folder called `build`?

Reply:
I thought it would be more organized, following the example in https://github.com/open-telemetry/opentelemetry-dotnet/tree/master/build. If it isn't of use I can remove that

Reply:
I would put build/ci scripts under one of the following folder:

![image](https://user-images.githubusercontent.com/17327289/93623583-ce027e00-f993-11ea-8753-b79968677a04.png)

Reply:
Sounds good. Should docfx.json and toc.yml also be relocated from the root directory?

Reply:
I think yes - unless the entire project is a document project (e.g. OpenTelemetry Specification).